community/portworx/values-metadata.yaml

```diff
@@ -0,0 +1,305 @@
+deploymentType:
+  __metadata:
+    name: "deploymentType"
+    label: "Deployment Type"
+    description: "Specifies where to install Portworx using Docker or OCI."
+    type: "string"
+    immutable: true
+    required: false
```

Comment:
@millerkc good point ! I actually want to hide this frmo the ICP UI. I tried setting `hidden: true` but even after that it still shows up. Am I missing something?

![screen shot 2018-09-17 at 1 17 15 pm](https://user-images.githubusercontent.com/26801153/45647988-24264900-ba7c-11e8-94a7-a3433c2bec8b.png)


I have this as below in values metadata file.

```
deploymentType:
  __metadata:
    name: "deploymentType"
    label: "Deployment Type"
    description: "Specifies where to install Portworx using Docker or OCI."
    type: "string"
    immutable: true
    hidden: true
    required: false

```

Reply:
@harsh-px sorry for the confusion here. You are using it correctly, but `hidden` has no effect in versions of ICP prior to 3.1, which was just released last week. Which ICP versions are you looking to support with this chart?

Reply:
I want to support whichever ICP version supports tiller 2.9.0 and above. 

Looks like my test system has ICP 2.1.0.3.

So if I want to hide these values in ICP prior to 3.1, I can simply remove them from values.yaml too.

Reply:
Right, the only way to hide them in prior versions would be to remove or comment them out.

ICP 3.1 is the only version that includes tiller 2.9.0. ICP 2.1.0.3 includes tiller 2.7.2, so this chart should not deploy on an out-of-the-box 2.1.0.3 environment. Did you upgrade the tiller in your environment?

Reply:
There's not a way to specify that in the chart, but in a typical ICP scenario, the user won't have access to deploy into kube-system, since our RBAC lets cluster administrators grant users or teams access to specific namespaces.

I'm not quite clear how you're deploying your chart on 2.1.0.3, but specifying a constraint of >=2.9.0. Are you using a non-default tiller?

Reply:
@millerkc Yes, I upgraded tiller in my ICP cluster to test the chart.